charts/aws-vpc-cni/templates/daemonset.yaml

```diff
@@ -116,9 +121,15 @@ spec:
         configMap:
           name: {{ include "aws-vpc-cni.fullname" . }}
 {{- end }}
+{{- if .Values.cri.enabled }}
+      - name: cri
+        hostPath:
+          path: {{- toYaml .Values.cri.hostPath | nindent 18 }}
+{{- else }}
```

Comment:
This is more like socket path override than enable. I'd say expose something like CRI override host path and if that's set, you override in the chart. You can even avoid having an explicit flag (i.e. `enabled: false`) to check this